### ISSUE TITLE
Crash while linking drawers. Cannot re-load world.

### ISSUE THREAD

**Issue by ciserman** [Buuz135/FunctionalStorage#58]
[crash-2022-07-20_11.24.21-server.txt](https://github.com/Buuz135/FunctionalStorage/files/9151547/crash-2022-07-20_11.24.21-server.txt)
While using the Linking Tool to configure my drawer setup my game crashed. Now I cannot load my world and need to restore a back up.

**Comment by Buuz135** [comment]
Can I get a pic of your setup?

**Comment by ciserman** [comment]
I've had to reload my world a few times, and have a configuration that seems to be working now, different than what I had. When it was crashing I just had one controller on the right and 4 drawers directly next to it in a line to the left, same y and x position. 
I was having trouble getting something particular to insert to the controller automatically, so I was trying to remove each drawer to re-add them, and when I clicked remove on one of the blocks with the Linking Tool that is when it crashed.
